README.md

```diff
@@ -299,6 +300,14 @@ Ender Dragon always spawns a new dragon egg
 - Suggested options: `true`, `false`
 - Categories: `SURVIVAL`, `FEATURE`, `REDCRAFT`
 
+## fastRedstoneCrafting
+
+Aggiunge crafting alternativi di dispenser e repeater
```

Comment:
Questo è il file inglese...

Adds fast redstone crafting for Dispenser and Repeater.